kube/services/arranger/README.md

```diff
@@ -0,0 +1,77 @@
+# TL;DR
+
+The arranger deployment provides a graphQL API endpoint that
+interacts with gen3 elastic search indexes in a way defined
+by an "arranger project" configuration saved in its own 
+elastic search indexes.
+
+## Arranger Project
+
+Gen3 uses the [arranger project](https://github.com/overture-stack/arranger) 
+to implement graphQL access to ES (elastic search) indexes generated form
+the graphdb by an ETL process.  The [gen3 arranger server](https://github.com/uc-cdis/gen3-arranger) publishes the graphQL endpoint for a single arranger project
+with arborist middlware for authz.
+
+## Arranger Development
+
+Vocabulary:
+* configuration index - an `arranger-projects-...` index generated by the arranger dashboard that holds arranger configurtion data
+* data index - a gen3 index generated via ETL from the graphdb
+
+Arranger deployment is a muiltistep process.  Develop and test the ETL and arranger configuration in a QA environment, then publish the tested configuration to production.
+
+* First - save an `etlMapping.yaml` to the gitops repo that configures the graphdb to ES ETL.
+* Next - run the ETL.  As a best practice generate versioned data indexes (`niad_v1`, `niad_v2`, ...), then alias the active index:
```

Comment:
at this point does the `tube` config in `manifest.json` need to be set up? My understanding is that that's where the index the ETL will point to is specified